models/House.js

```diff
@@ -1,29 +1,73 @@
-const mongoose = require("mongoose");
+const mongoose = require('mongoose');
 const Schema = mongoose.Schema;
-const User = require("./User");
-const Bill = require("./Bill");
+const User = require('./User');
+const Bill = require('./Bill');
 
 const houseSchema = new Schema({
-  owner: { type: Schema.ObjectId, ref: "User" },
+  owner: {
+    type: Schema.ObjectId,
+    ref: "User"
+  },
   account: {
-    bsb: Number,
-    accNumber: Number
+    bsb: {
+      type: Number,
+      required: [true, 'BSB number required'],
+      validator: function (v) {
+        return /\d{2} \d{4}/.test(v);
+      },
+      message: props => `${props.value} is not a valid BSB number!`
+
+    },
+    accNumber: {
+      type: Number,
+      required: [true, 'Account number required'],
+      validator: function (v) {
+        return /\d{4} \d{4}/.test(v);
```

Comment:
Same goes here, do you prefer to have space / dash as separator ? @demounit

Reply:
Prefer dash I guess

Reply:
Same here